src/Playwright.MSTest/ContextTest.cs

```diff
@@ -40,6 +40,10 @@ public async Task ContextSetup()
 
     public virtual BrowserNewContextOptions ContextOptions()
     {
-        return null!;
+        return new()
+        {
+            Locale = "en-US",
```

Comment:
This seems irrelevant to .net 8

Reply:
Discussed offline: Align it with the upstream defaults.